### ISSUE TITLE
Link to updated working plugin (homebridge-shortcuts-buttons)

### ISSUE THREAD

**Issue by duddu** [kylewhirl/homebridge-shortcuts#3]
Hi @kylewhirl,

I really liked the idea of this plugin so, since it's marked as not-working, I decided to work on a plugin myself that executes apple shortcuts from homebridge (with the little addition of executing a callback command or shortcut after completion via x-callback-url): [duddu/homebridge-shortcuts-buttons](https://github.com/duddu/homebridge-shortcuts-buttons).

I wondered if you would consider maybe linking to this plugin from your readme for a working version of the same functionality - as your repo already has some visibility? (I know it's a lot to ask - but worth a shot!)

Also, keen to get your feedback on my code if you fancy, as you likely encountered similar problems whilst developing your plugin.

In the meantime, thanks for your time!

**Comment by kylewhirl** [comment]
Yeah of course I can! I checked out your plugin and you did a great job! It’s exactly what I envisioned for a shortcuts pluginOn May 3, 2024, at 10:35 AM, Davide Doronzo ***@***.***> wrote:﻿
Hi @kylewhirl,
I really liked the idea of this plugin so, since it's marked as not-working, I decided to work on a plugin myself that executes apple shortcuts from homebridge (with the little addition of executing a callback command or shortcut after completion via x-callback-url): duddu/homebridge-shortcuts-buttons.
I wondered if you would consider maybe linking to this plugin from your readme for a working version of the same functionality - as your repo already has some visibility? (I know it's a lot to ask - but worth a shot!)
Also, keen to get your feedback on my code if you fancy, as you likely encountered similar problems whilst developing your plugin.
In the meantime, thanks for your time!
